api/base/urls.py

```diff
@@ -1,17 +1,17 @@
-from django.conf import settings as drf_settings
 from django.conf.urls import include, url
 from django.conf.urls.static import static
 from django.views.generic.base import RedirectView
 
 from . import views
 from . import settings
 from . import versioning
+from api.base.swagger import get_swagger_view
 
 default_version = versioning.decimal_version_to_url_path(settings.REST_FRAMEWORK['DEFAULT_VERSION'])
 
 # Please keep URLs alphabetized for auto-generated documentation
 urlpatterns = [
-    url('^(?P<version>(v2))/',
+    url(r'^(?P<version>(v2))/',
```

Comment:
The `WIP` note can be resolved by replacing the parenthesized option with the standard url param convention (`r'^(?P<version>\w+)/'`), but this may have unintended consequences.